Sources/SwiftGRPCNIOTestData/GRPCSwiftCertificate.swift

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import NIOSSL
+
+/// Wraps `NIOSSLCertificate` to provide the certificate common name and expiry date.
+public struct GRPCSwiftCertificate {
+  public var certificate: NIOSSLCertificate
+  public var commonName: String
+  public var notAfter: Date
+
+  public static func ca() throws -> GRPCSwiftCertificate {
```

Comment:
make these static properties that only get initialized once instead?

Reply:
We'd have to `try!` if they were static properties; I _guess_ this isn't so much an issue as we know it works with the current certificates. What do you think?

Reply:
I think the `try!` should be fine here.